### PR TITLE
Added Vial Support for K3 Pro ANSI

### PR DESCRIPTION
**Pull request by jSQrD-dev**
Added Vial Support for K3 Pro ANSI

### CODE REVIEW

File: keyboards/keychron/k3_pro/ansi/white/keymaps/vial/rule.mk [hunk 1]
```diff
@@ -0,0 +1,2 @@
+VIA_ENABLE = yes
+VIAL_ENABLE = yes
```
Comment (adophoxia) [inline keyboards/keychron/k3_pro/ansi/white/keymaps/vial/rule.mk]:
```suggestion

```
Invalid file. Should be removed. Not sure why it's there.

File: keyboards/keychron/k3_pro/ansi/rgb/keymaps/vial/config.h [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+#define VIAL_UNLOCK_COMBO_ROWS \
+    { 0, 3 }
+
+#define VIAL_UNLOCK_COMBO_COLS \
+    { 0, 13 }
+
+#define RGB_MATRIX_FRAMEBUFFER_EFFECTS
+
+#define RGB_MATRIX_KEYPRESSES
```
Comment (adophoxia) [inline keyboards/keychron/k3_pro/ansi/rgb/keymaps/vial/config.h]:
```suggestion

```
Already exists at [k3_pro/ansi/rgb/config.h](https://github.com/adophoxia/vial-qmk/blob/vial-keychron-bluetooth-playground/keyboards/keychron/k3_pro/ansi/rgb/config.h#L51-L52)

File: keyboards/keychron/k3_pro/ansi/white/keymaps/vial/vial.json [hunk 3]
```diff
@@ -0,0 +1,253 @@
+{
+    "name": "Keychron K3 Pro ANSI White",
+    "vendorId": "0x3434",
+    "productId": "0x0233",
+    "lighting": "qmk_backlight",
```
Comment (adophoxia) [inline keyboards/keychron/k3_pro/ansi/white/keymaps/vial/vial.json]:
```suggestion
    "lighting": "none",
```
This variant uses LED_Matrix, and there's no lighting option for it for `vial.json`.

File: keyboards/keychron/k3_pro/ansi/white/keymaps/vial/config.h [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+#define VIAL_KEYBOARD_UID \
+    {0x60, 0xC5, 0xB1, 0xE4, 0x61, 0x9D, 0xBE, 0x9D}
+
+#define VIAL_UNLOCK_COMBO_ROWS \
+    { 0, 3 }
+
+#define VIAL_UNLOCK_COMBO_COLS \
+    { 0, 13 }
+
+#define BACKLIGHT_BREATHING
```
Comment (adophoxia) [inline keyboards/keychron/k3_pro/ansi/white/keymaps/vial/config.h]:
```suggestion

```
Already exists as a lighting effect for [LED Matrix](https://github.com/adophoxia/vial-qmk/blob/vial-keychron-bluetooth-playground/keyboards/keychron/k3_pro/ansi/white/config.h#L46).

**Review by adophoxia — CHANGES_REQUESTED** [review]

**Comment by jSQrD-dev** [comment]
All changes have been made.

**Review by adophoxia — APPROVED** [review]

**Pull request merged**